quickstart/comparisons/cloud_templates.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Cloud Templates (AWS CloudFormation, etc)
+---
+
+All major cloud providers offer their own form of infrastructure-as-"code" by way of JSON or YAML-based templating
+solutions. This includes
+
+* AWS CloudFormation
+* Azure Resource Manager
+* Google Deployment Manager
+
+These configuration files are often uploaded to a hosted service in the target cloud, where it will then process the
+files to create, update, or delete resources as needed.
+
+Pulumi's model shares a lot with these systems. Although Pulumi programs are written in imperative, familiar languages,
+they are ultimately evaluated to produce a similar set of create, update, or delete operations for your cloud resources.
+
+This is where Pulumi begins to diverge from these other solutions, however.
+
+Pulumi lets you use your favorite languages. Instead of bespoke templating solutions, often tacked on to a markup
+in an awkward and hard-to-learn and -remember way, Pulumi programs use real code. Thanks to simple things like
```

Comment:
did you want all dashes until remember or just have one extra?

Reply:
I did, as I meant hard-to-learn and hard-to-remember, but I'll admit this is an awkward phrasing. I can look into tidying it up.